### PR TITLE
Css ecommerce page-Dulce Herrera

### PR DESCRIPTION
**Pull request by dulceherrera**
Screen Shot 2022-07-09 at 3 33 51 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Dulce, the Yves Saint Laurent text is not aligned with the text of the sibling elements, please revise and submit a new screenshot with your revision.

**Comment by dulceherrera** [comment]
Screen Shot 2022-07-11 at 11 15 21 AM

File: css-ecommerce-page/styles.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-fourth {
+  flex-basis: 25%;
+}
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css:16]: There are five elements next to one another, that is a fifth, not a fourth 🤔

File: css-ecommerce-page/styles.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+.a,
+.b,
+.c,
+.d,
+.e {
+  display: inline;
+  color: black;
+}
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css:51]: These class names are not descriptive at all. Could you make this one meaningfully-named class that you could apply to all these elements?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-ecommerce-page/styles.css (hunk 2 above)
Comment (dulceherrera) [reply css-ecommerce-page/styles.css]: Screen Shot 2022-07-15 at 10 37 03 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**